### PR TITLE
⬆️: Upgrading Flow to 0.61

### PR DESCRIPTION
**Pull request by wldcordeiro**
Associated Issue: N/A

### Summary of Changes

* Upgrade Flow to 0.61.0 

### Test Plan

* Ran test suite
* Ran through app scenarios (stepping, breakpoints, etc to ensure no odd regressions)

### CODE REVIEW

File: src/reducers/pending-breakpoints.js [hunk 1]
```diff
@@ -81,12 +81,16 @@ function addBreakpoint(state, action) {
     return state;
   }
 
-  // when the action completes, we can commit the breakpoint
-  const { value: { breakpoint } } = action;
-  const locationId = makePendingLocationId(breakpoint.location);
-  const pendingBreakpoint = createPendingBreakpoint(breakpoint);
+  if (action.type === "ADD_BREAKPOINT") {
```
Comment (wldcordeiro) [inline src/reducers/pending-breakpoints.js]: Not sure about this change yet. Somewhat related to the remaining error. Just involves some type refinements in certain places.

**Comment by jasonLaster** [comment]
Nice!

File: src/reducers/pending-breakpoints.js (hunk 1 above)
Comment (wldcordeiro) [reply src/reducers/pending-breakpoints.js]: I noticed this function is called for both add and enable breakpoints but I don't see the `value` property on the enable breakpoint action @jasonLaster Could you help with a little clarification?

**Comment by wldcordeiro** [comment]
`flow-bin` 0.62 seems to be a little delayed so I'm removing the WIP and making this just the upgrade to 0.61 and will follow-up with the next one.

**Pull request merged**